src/components/global/Footer.jsx

```diff
@@ -1,8 +1,208 @@
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Link from '@mui/material/Link';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import { useTheme } from '@mui/material/styles';
+// Other Library Imports
+import dayjs from 'dayjs';
+import { FaDiscord, FaGithub, FaLinkedin, FaMeetup } from 'react-icons/fa6';
+// Image Imports
+import roseLogo from '/assets/rose_logo.png';
+import logoBlob from '/assets/logoBlob.svg';
+
+// Legal links
+const legalLinks = [
+  {
+    href: 'https://www.codepdx.org/',
+    title: 'About'
+  },
+  {
+    href: 'https://www.codepdx.org/',
+    title: 'Code of Conduct'
+  },
+  {
+    href: 'https://www.codepdx.org/',
+    title: 'Contact Us'
+  },
+  {
+    href: 'https://www.codepdx.org/',
+    target: '_blank',
+    rel: 'noopener',
+    ml: 0.5,
+    text: `©${dayjs().year()}`,
+    title: 'CODE PDX'
+  }
+];
+
+// Social media links
+const socialMediaLinks = [
+  {
+    href: 'https://github.com/codeforpdx/',
+    icon: <FaGithub size={45} />,
+    target: '_blank',
```

Comment:
none of these need the `target` or `rel` since they all have it, it can just be defined in the `.map()` - which it already is. So, these object properties are not being used at all.

Reply:
removed from individual icons and left in mapping.